Assets/SEE/Controls/SEEInput.cs

```diff
@@ -29,7 +57,7 @@ public static class SEEInput
         /// <returns>true if the user requests this action and <see cref="KeyboardShortcutsEnabled"/></returns>
         public static bool Help()
         {
-            return KeyboardShortcutsEnabled && Input.GetKeyDown(KeyBindings.Help);
+            return KeyboardShortcutsEnabled && Input.GetKeyDown(bindings["Help"]);
```

Comment:
Do not use a string literal if it occurs more than once. Use a named string constant instead.

This should be done for all all other string literals here and in `KeyBindings.cs`. Define those string contants in `KeyBindings.cs`.